gradle/projects.main.properties

```diff
@@ -6,6 +6,7 @@ nessie-compatibility-tests=compatibility/compatibility-tests
 nessie-compatibility-jersey=compatibility/jersey
 nessie-events-api=events/api
 nessie-events-quarkus=events/quarkus
+nessie-events-ri=events/ri
```

Comment:
Not in the nessie-bom?

Reply:
None of the event modules is declared in `:nessie-bom`... I guess that's an oversight.